test/Lachain.CoreTest/RPC/HTTP/Web3/AccountServiceWeb3Test.cs

```diff
@@ -115,6 +117,19 @@ public void Test_GetBalance()
             Assert.AreEqual(balNew, "0x115557b419c5c1f3fa018400000000");
         }
 
+        [Test]
+        // Changed GetBalance to public
+        public void Test_GetBalance_pending()
```

Comment:
Let's also keep the function name consistent with the rest. 
"Test_GetBalancePending"